src/arb.rs

```diff
@@ -10,20 +10,25 @@ use rand::{
 };
 
 use arbitrary::{size_hint, Unstructured};
+use rand::RngCore;
+use rand::SeedableRng;
 
 use crate::{CidGeneric, Version};
 
 impl quickcheck::Arbitrary for Version {
-    fn arbitrary<G: Gen>(g: &mut G) -> Self {
-        let version = if g.gen_bool(0.7) { 1 } else { 0 };
+    fn arbitrary(g: &mut Gen) -> Self {
+        // XXX: Adding a bias towards version 1 isn't vital.
+        // let mut rng = rand::rngs::SmallRng::seed_from_u64(u64::arbitrary(g));
+        // let version = if rng.gen_bool(0.7) { 1 } else { 0 };
```

Comment:
Let's get rid of this comment. XXX usually means "don't merge".

Reply:
removed